src/components/Dashboard/Dashboard.jsx

```diff
@@ -41,7 +40,11 @@ export const Dashboard = props => {
 
   return (
     <Container fluid>
-      <PopUpBar />
+      {props.match.params.userId && props.auth.user.userid !== props.match.params.userId ? (
+        <PopUpBar />
+      ) : (
+        ''
+      )}
```

Comment:
Please assign the boolean expression on line 43 as its own const with a descriptive name before the return and use that instead.